admin-manual/installation/installation.rst

```diff
@@ -317,6 +317,20 @@ Update your system
 .. code:: bash
 
    sudo yum update
+   
+If your environment uses SELinux, you'll need to tweak some configurations:
```

Comment:
Suggest adding a sentence after this

`At a minmum you will need to run the following commands.  Additional configuration may be required for your local setup.  `